tests/Fixtures/Integration/priority/no_duplicate_semicolons,switch_case_semicolon_to_colon.test

```diff
@@ -1,17 +0,0 @@
---TEST--
-Integration of fixers: no_duplicate_semicolons,switch_case_semicolon_to_colon.
```

Comment:
has been done;
https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/master/tests/Fixtures/Integration/priority/no_empty_statement,switch_case_semicolon_to_colon.test

this one `tests/Fixtures/Integration/priority/no_duplicate_semicolons,combine_consecutive_unsets.test` does need a replacement indeed :+1: